api/base/settings/__init__.py

```diff
@@ -11,19 +11,27 @@
 import itertools
 
 from .defaults import *  # noqa
+from website import settings
 
 try:
     from .local import *  # noqa
 except ImportError as error:
     warnings.warn('No api/base/settings/local.py settings file found. Did you remember to '
                   'copy local-dist.py to local.py?', ImportWarning)
 
+if not settings.DEV_MODE:
```

Comment:
This might break the admin module -- it should probably be checking `os.environ.get('DJANGO_SETTINGS_MODULE')` as below.